crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -36,6 +36,12 @@ async fn local_node_mixed_limit_and_market_orders() {
     run_test(mixed_limit_and_market_orders_test).await;
 }
 
+#[tokio::test]
+#[ignore]
+async fn local_node_replace_order() {
```

Comment:
I think this test can live in its own file since the other tests are more concerned with verifying various settlement behaviors.